docs/api/js/features.md

```diff
@@ -21,61 +21,78 @@ In order to make the SDK as flexible as possible, the library uses the overrides
 ```typescript
 {
     customData: {
-        feeToken?: Address;
         ergsPerPubdata?: BigNumberish;
         factoryDeps?: BytesLike[];
+        customSignature?: BytesLike;
+        paymasterParams?: {
+            paymaster: Address;
+            paymasterInput: BytesLike;
+        };
     }
 }
 ```
 
 Examples:
 
-Override to pay fees in the token with the address `0xd35cceead182dcee0f148ebac9447da2c4d449c4`:
-
-```typescript
-{
-  customData: {
-    feeToken: "0xd35cceead182dcee0f148ebac9447da2c4d449c4";
-  }
-}
-```
-
-Override to withdraw ETH and pay fees in the token with the address `0xd35cceead182dcee0f148ebac9447da2c4d449c4`:
-zzz
+Override to deploy a contract with bytecode `0xcde...12` and enforce that the operator will not charge more than `100` ergs per published bytes on layer 1:
 
 ```typescript
 {
     customData: {
-        feeToken: "0xd35cceead182dcee0f148ebac9447da2c4d449c4",
-        withdrawToken: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
+        ergsPerPubdata: "100",
+        factoryDeps: ["0xcde...12"],
     }
 }
 ```
 
-Override to deploy a contract with bytecode `0xcde...12` and enforce that the operator will not charge more than `100` ergs per published bytes on layer 1:
+Override and use custom signature `0x123456` for account, while using paymaster with address `0x8e1DC7E4Bb15927E76a854a92Bf8053761501fdC` and paymaster input `0x8c5a3445`:
 
 ```typescript
 {
-    // feeToken is not supplied, using ETH by default
     customData: {
-        ergsPerPubdata: "100",
-        factoryDeps: ["0xcde...12"],
+        customSignature: "0x123456",
+        paymasterParams: {
+            paymaster: "0x8e1DC7E4Bb15927E76a854a92Bf8053761501fdC",
+            paymasterInput: "0x8c5a3445"
+        }
     }
 }
 ```
 
+## Encoding paymaster params
+
+While the paymaster feature by itself does not impose any limitations on values of the `paymasterInput`, the Matter Labs team endorses certain types of [paymaster flows](../../dev/zksync-v2/aa.md#built-in-paymaster-flows) that are processable by EOAs.
```

Comment:
as above: zksync / ML

Reply:
Does it matter? I think it is ok to use both interchangeably

Reply:
> Does it matter? 

Probably not, but I like us to be consistent. Also, while not really my reasoning, I think that we aim to focus on zkSync and not the company behind it (e.g. we stopped tweeting from the ML account)

Reply:
I think this is to promote the platform, but in the case of documentation the user is already interested enough to read through it, so I think it is ok